### DISCUSSION TITLE
Zero knowledge of coding

### DISCUSSION THREAD

**Discussion by omarelsobky** [rawandahmad698/PyChatGPT#122] [Q&A]
Should I have an account on ChatGPT first to access it through your method?
if not, then how to start using ChatGPT on the cmd after installing? 
Excuse my naive question; as I mentioned I have zero knowledge of how coding works and want to use ChatGPT but it is not available in my country.

**Comment by jonasjancarik** [comment]
In short, I believe you need an account. If you can't use your phone number to get a confirmation text message, maybe try Firefox Relay (which [should](https://blog.mozilla.org/en/privacy-security/hide-your-phone-number-with-firefox-relay/) offer "burner" numbers - but I am not sure if that service is available everywhere either) or try getting a US [number with Skype](https://www.skype.com/en/features/online-number/).

By the way, I don't think this library works at the moment, given the message at the top of the [Readme](https://github.com/rawandahmad698/PyChatGPT#readme). I would suggest trying another library like https://github.com/acheong08/ChatGPT, although if you have literally no coding skills it may be a bit tricky to figure out.
